registry.json

```diff
@@ -3914,5 +3914,16 @@
     "telegram": "lukas_psya",
     "operatorName": "",
     "github": "rafek47"
+  },
+    {
+    "ticker": "OTG",
+    "poolId": "c825168836c5bf850dec38567eb4771c2e03eea28658ff291df768ae",
+    "social": {
+      "website": "https://adamantium.online/",
+      "twitter": "OTG_WCat"
+    },
+    "telegram": "WCat13",
+    "operatorName": "WCat",
```

Comment:
Please can you change this to either your real full name, real first name or leave blank?